src/css-grids/cssgrids-style.css

```diff
@@ -0,0 +1,427 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-image: url(https://images.unsplash.com/photo-1534543210152-32025bcfaad9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80);
```

Comment:
add quotes around the link to the image.

i didn't realize the bg was an image! Thought you used a gradient. I wonder if you could recreate this gradient yourself. and add a round div at the bottom to replicate the sun which also has a gradient... could be fun! (not necessary) 
check these out: 

- [https://gradientlab.space/](url)
-[https://cssgradient.io/](url)